src/views/ManageSites.vue

```diff
@@ -116,26 +116,30 @@
             </div>
           </div>
         </li>
+        <li>
```

Comment:
@johncrossman does it make sense to have this button inside a list item? JAWS  will see the <ul> and say "list of 4 items" when there are actually 3 options.

Reply:
@lyttam - Good point. I'll amend tomorrow morning. Happy Halloween! 👻

Reply:
@lyttam  - PR amended.